spectree/plugins/falcon_plugin.py

```diff
@@ -22,6 +31,28 @@
 from spectree.utils import cached_type_hints
 
 
+class StreamWrapper:
+    def __init__(self, stream):
+        self._buf = CachedFile()
```

Comment:
I'm a bit leery of replacing a body part's `.stream` attribute, but I guess we don't offer easy workarounds for that atm...

How are these files cleaned up?

As I understand at least on Linux, they are created using the `O_TMPFILE` flag which makes them go away when the handle is closed. However, that only happens when the part object is garbage collected, since we are not closing anything explicitly.

Maybe it is good enough? :thinking:

Reply:
Yeah. I have to replace the `.stream` and provide a similar interface.

The files are cleaned by garbage collection. I didn't find a way to close it explicitly.